web-bluetooth/service-worker.js

```diff
@@ -1,20 +1,27 @@
-self.addEventListener('fetch', function(event) {
-  const request = event.request;
-  const url = new URL(event.request.url);
+function addToCache(request, networkResponse) {
+  return caches.open('web-bluetooth')
+    .then((cache) => cache.put(request, networkResponse.clone()));
+}
 
-  // Don't cache anything that is not on this origin.
-  if (url.origin !== location.origin) {
-    return;
-  }
+function getCacheResponse(request) {
+  return caches.open('web-bluetooth').then((cache) => {
```

Comment:
We've been (fairly, I think, for some definition of fairly) consistent about not putting `()` around single-parameter arrow functions in GoogleChrome/samples.

Could you rewrite this to `then(cache => {`, and make that change elsewhere as well?

(I need to get a rule for http://eslint.org/docs/rules/arrow-parens added to our standard `eslint` config.)